repo/packages/J/jenkins/9/package.json

```diff
@@ -1,20 +1,20 @@
 {
   "packagingVersion": "2.0",
   "name": "jenkins",
-  "version": "0.2.3",
+  "version": "1.0.0-1.651.3",
   "scm": "https://github.com/mesosphere/jenkins-mesos.git",
   "maintainer": "support@mesosphere.io",
-  "website": "https://jenkins-ci.org",
+  "website": "https://jenkins.io",
```

Comment:
(Bearing in mind that this is where the docs currently on the jenkins-mesos repo will be moved to.)